src/ServiceManagement/Services/Commands.Test/Websites/PublishAzureWebsiteProjectTests.cs

```diff
@@ -18,6 +17,57 @@ public class PublishAzureWebsiteProjectTests : WebsitesTestBase
     {
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void PublishFromPackageWithoutSetParameters()
+        {
+            const string websiteName = "test-site";
+            const string package = "test-package";
+            string slot = null;
+            var connectionStrings = new Hashtable { ["DefaultConnection"] = "test-connection-string" };
```

Comment:
@eocrawford This is where the compilation issue is, according to the log.

I believe you use the same literal construction as a dictionary, i.e.:

```c#
var connectionStrings = new HashTable {
    {"DefaultConnection", "test-connection-string"}
};
```